packages/rocket-uploads3-store-event-aws-infrastructure/.npmignore

```diff
@@ -0,0 +1 @@
+!src/lambdas/node_modules
```

Comment:
@rdoria1 now thinking about this, I don' know if the github action can publish your rocket to the npm registry so maybe this won't work 100% as expected. For example the static rocket is published: https://www.npmjs.com/package/@boostercloud/rocket-static-sites-aws-infrastructure

Reply:
Also for those others looking at this PR, this `.npmignore` file is set because the previous `.gitignore` (root folder) isn't allowing us to publish the lambda `node_modules`. We tried with another inner `.gitignore` and other solutions but it wasn't possible. We need to use `.npmignore` to whitelist the lambda node modules and publish using npm instead of yarn 😢

Reply:
yes I think you are right. As of now it won't compile because it can't find the nodes modules of the lambda. I think I need to add it OR I need to add an action form the rocket so that it it install the node modules of the lambda before compiling the rocket. What do you think? @juanjoman

Reply:
Yeah for this case I think you can set the compile script like: `yarn compile <lambda_folder> && cp -R <origin> <destiny>`. The thing is that `lerna run compile` is running a `yarn compile` on the root of the rocket @rdoria1 .

Reply:
Yeah actually I'll add 
`"postinstall": "yarn --cwd src/lambdas install",` to the package.json rocket and I think it will work :)

Reply:
@juanjoman @javiertoledo I already rebase the branch with the newest changes and even with npm we have the same problem,the lambda folder won't compile nor install any dependencies. I think this is expected. The lambda folder is a sub folder of the package and by default it will only look into the package.json of the root folder. 

I added `"postinstall": "npm --prefix src/lambdas install",`

Reply:
@rdoria1 @juanjoman Would this bug be automatically fixed if the rocket were in a separate repo?

Reply:
@alvaroloes I didn't get that one. 

So we have an embedded package.json for the lambda we are defining which is why we need to run an npm install on that subfolder as well (not only the main one for the rocket) 
I think this solution is fine. Thoughts?

Reply:
Oh I think @alvaroloes's question is if we would need this `.npmignore` in an external repo. I think we could avoid it if we specify on the `.gitignore` that we specifically don't want the root `node_modules` right? @rdoria1